offline/upgrading.md

```diff
@@ -11,7 +11,40 @@ $ cd ...  # you pick a good location!
 Obtain the latest airgrap artifact for wire-server-deploy. Please contact us to get it for now. We are
 working on publishing a list of airgap artifacts.
 
-Extract the above listed artifacts into your workspace:
+## Clean up enough disk space to operate:
+
+### AdminHost
+prune old containers that are generated during our 'd' invocations:
+```
+df
+sudo docker container prune
+```
+
+### Kubernetes hosts:
+Remove wire-server images from two releases ago, or from the current release that we know are unused. for instance, 
+
+```
+sudo docker image ls
+sudo docker image ls | grep -E "^quay.io/wire/" | grep "2.106.0" | sed "s/.*[ ]*\([0-9a-f]\{12\}\).*/sudo docker image rm \1/"
```

Comment:
```suggestion
VERSION="2.106.0"
sudo docker image ls | grep -E "^quay.io/wire/" | grep $VERSION | sed "s/.*[ ]*\([0-9a-f]\{12\}\).*/sudo docker image rm \1/"
```

Give a name to the magic number. Not all customers might know our release naming schema and immediately recognize it.